Makefile

```diff
@@ -33,9 +33,7 @@ export DOCKER_DEST
 COVERPROFILES := \
 	backend-coverage.coverprofile \
 	config-coverage.coverprofile \
-	context-coverage.coverprofile \
-	image-coverage.coverprofile \
-	metrics-coverage.coverprofile
+	image-coverage.coverprofile
```

Comment:
Correct, the tests were the stub tests and after removing them, there were no more tests (so `make test` broke since it couldn't find those files).